ExportCSV.ascx

```diff
@@ -1,34 +1,58 @@
+
+<dnn:DnnCssInclude runat="server" FilePath="~/Resources/Shared/components/TimePicker/Themes/jquery-ui.css" />
+<dnn:DnnCssInclude runat="server" FilePath="~/Resources/Shared/components/TimePicker/Themes/jquery.ui.theme.css" />
+
 <div class="dnnForm">
-<div class="dnnFormItem">
-    <dnn:Label ID="plFolder" runat="server" ControlName="cboFolders" Suffix=":" />
-    <asp:DropDownList ID="cboFolders" runat="server" CssClass="NormalTextBox" Width="300" />
-</div>
-<div class="dnnFormItem">
-    <dnn:Label ID="plFile" runat="server" ControlName="txtFile" Suffix=":" />
-    <asp:TextBox ID="txtFile" CssClass="NormalTextBox" runat="server" MaxLength="200"
-        Width="300" />
-</div>
-<div class="dnnFormItem">
-    <dnn:label id="plDelimiter" runat="server" controlname="rblDelimiter" suffix=":" />
-    <asp:RadioButtonList ID="rblDelimiter" runat="server"  RepeatDirection="Horizontal"
-        CssClass="dnnFormRadioButtons ">
-        <asp:ListItem Selected="True" Value="," resourcekey="comma"></asp:ListItem>
-        <asp:ListItem Value=";" resourcekey="semicolon"></asp:ListItem>
-    </asp:RadioButtonList>
-</div>
-<div class="dnnFormItem">
-    <dnn:label id="plSystemFields" runat="server" controlname="cbSystemFields"  />
-    <asp:CheckBox ID="cbSystemFields" runat="server"/>  
+    <div class="dnnFormItem">
+        <dnn:Label ID="plFolder" runat="server" ControlName="cboFolders" Suffix=":" />
+        <asp:DropDownList ID="cboFolders" runat="server" CssClass="NormalTextBox" Width="300" />
+    </div>
+    <div class="dnnFormItem">
+        <dnn:Label ID="plFile" runat="server" ControlName="txtFile" Suffix=":" />
+        <asp:TextBox ID="txtFile" CssClass="NormalTextBox" runat="server" MaxLength="200"
+            Width="300" />
+    </div>
+    <div class="dnnFormItem">
+        <dnn:label id="plDelimiter" runat="server" controlname="rblDelimiter" suffix=":" />
+        <asp:RadioButtonList ID="rblDelimiter" runat="server" RepeatDirection="Horizontal"
+            CssClass="dnnFormRadioButtons ">
+            <asp:ListItem Selected="True" Value="," resourcekey="comma"></asp:ListItem>
+            <asp:ListItem Value=";" resourcekey="semicolon"></asp:ListItem>
+        </asp:RadioButtonList>
+    </div>
+    <div class="dnnFormItem">
+        <dnn:label id="plSystemFields" runat="server" controlname="cbSystemFields" />
+        <asp:CheckBox ID="cbSystemFields" runat="server" />
+    </div>
+    <div class="dnnFormItem">
+        <dnn:label id="plInitialDate" runat="server" controlname="cbSystemFields" />
+        <asp:TextBox ID="txtInitialDate" CssClass="NormalTextBox" runat="server" MaxLength="200"
+            Width="300" />
+    </div>
+    <div class="dnnFormItem">
+        <dnn:label id="plFinalDate" runat="server" controlname="cbSystemFields" />
+        <asp:TextBox ID="txtFinalDate" CssClass="NormalTextBox" runat="server" MaxLength="200"
```

Comment:
There's no validation to prevent errors and user experience issues. Also, the `readonly` attribute should probably be here and not only in the JavaScript, so that it's still `readonly` if JS is turned off - unless you have a technical reason for that. :)